metricflow/model/semantics/linkable_spec_resolver.py

```diff
@@ -476,7 +504,19 @@ def __init__(
                     assert_values_exhausted(metric.type)
 
             self._metric_to_linkable_element_sets[metric.name] = linkable_sets_for_measure
-        logger.info(f"Building the [metric -> valid linkable element] index took: {time.time() - start_time:.2f}s")
+
+        # If no metrics are specified, the query interface supports distinct dimension values from a single semantic
+        # model.
```

Comment:
Is this true? Because build_plan_for_distinct_values doesn't behave as if this is true. For that matter, neither does this code. Maybe we just remove this comment?

Reply:
Actually, it does:

https://github.com/dbt-labs/metricflow/blob/868748ae7f16ddf2479594b7ea43d9a33759c4e4/metricflow/test/query_rendering/test_query_rendering.py#L196